tests/integration-tests/tests/common/utils.py

```diff
@@ -58,15 +56,5 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
 
 
 @retry(stop_max_attempt_number=3, wait_fixed=5000)
-def fetch_instance_slots(region, instance_type, slots="vcpus"):
```

Comment:
Note that the new version of this function does not support the `slots` kwarg. This was done intentionally. None of the uses of this function appear anything other than the default value:

```bash
➜  pwd
/Users/tilne/repos/aws-parallelcluster/tests/integration-tests
➜  egrep -nir --exclude '*.pyc' 'fetch_instance_slots' . | grep -v import
./tests/update/test_update.py:66:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/intel_hpc/test_intel_hpc.py:29:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/disable_hyperthreading/test_disable_hyperthreading.py:29:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/disable_hyperthreading/test_disable_hyperthreading.py:45:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/common/utils.py:59:def fetch_instance_slots(region, instance_type):
./tests/scaling/test_mpi.py:29:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/dns/test_dns.py:32:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/efa/test_efa.py:37:    slots_per_instance = fetch_instance_slots(region, instance)
./tests/efa/test_efa.py:64:    slots_per_instance = fetch_instance_slots(region, instance)
```